site/content/docs/example-ckan.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Example: data catalog with data coming from CKAN"
+authors: ['Luccas Mateus']
```

Comment:
@demenech I think we should use id not name of author because this is needed to retrieve author info such as avatar and full name.

Reply:
Based on this https://github.com/datopian/portaljs/blob/main/site/lib/getAuthorsDetails.ts#L21 name should work as well:

```typescript
  return blogAuthors.map((author) => {
    const person = allPeopleMetadata.find(
      ({ id, name }) => id === author || name === author
    );

    return person ?? { name: author, avatar: siteConfig.avatarPlaceholder };
  });
```

And it's retrieving the avatar correctly e.g https://portaljs.org/blog/example-data-catalog so I think it's working as expected